packages/client-telegram/src/messageManager.ts

```diff
@@ -660,7 +661,7 @@ export class MessageManager {
                     this.runtime.character.templates
                         ?.telegramShouldRespondTemplate ||
                     this.runtime.character?.templates?.shouldRespondTemplate ||
-                    composeRandomUser(telegramShouldRespondTemplate, 2),
```

Comment:
Could you clarify the reasoning behind removing the `composeRandomUser` function? I'm curious about the context for this change and whether there's an alternative approach we should be using.